docs/reference/glossary.md

```diff
@@ -45,6 +45,15 @@ Show page templates (`show.html`), do have access to their parent index page's `
 
 There is a single doc with the slug, `global`, which is always loaded and available to page templates as `data.global`. This is useful for shared, site-wide headers and footers that are editable, etc. It is managed by the `@apostrophecms/global` module.
 
+## Slug
+
+The slug of an Apostrophe doc is a string that uniquely identifies that doc within the database. Apostrophe will enforce that no two docs may have the same slug. Unlike the `_id` property of the database document, slugs _can_ change, however. Apostrophe user interface and REST API will suggest slugs automatically based on the document's title on creation.
+
+If the doc has a dedicated page (or [show page](#show-page) for a piece), the slug is used as a component of the page URL. This works differently for pages and pieces.
```

Comment:
If the doc is a page, or a piece with a show page...

Reply:
better.